dali/python/backend_impl.cc

```diff
@@ -81,6 +90,12 @@ py::dict ArrayInterfaceRepr(Tensor<Backend> &t) {
   tup[0] = py::reinterpret_borrow<py::object>(PyLong_FromVoidPtr(t.raw_mutable_data()));
   tup[1] = true;
   d["data"] = tup;
+  if (std::is_same<Backend, GPUBackend>::value) {
+    d["version "] = 2;
+  } else {
+    d["version "] = 3;
```

Comment:
can you explain the meaning behind those versions? And why is there a space in the key?

Reply:
same question here :+1:

Reply:
Added comment with the reference of the spec

Reply:
This doesn't explain why there's trailing space in the name `"version "`

Reply:
Typo. Will fix.